tudatBundle/pagmo2/include/pagmo/algorithms/GACO.hpp

```diff
@@ -0,0 +1,1359 @@
+/* Copyright 2017 PaGMO development team
+
+This file is part of the PaGMO library.
+
+The PaGMO library is free software; you can redistribute it and/or modify
+it under the terms of either:
+
+  * the GNU Lesser General Public License as published by the Free
+    Software Foundation; either version 3 of the License, or (at your
+    option) any later version.
+
+or
+
+  * the GNU General Public License as published by the Free Software
+    Foundation; either version 3 of the License, or (at your option) any
+    later version.
+
+or both in parallel, as here.
+
+The PaGMO library is distributed in the hope that it will be useful, but
+WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received copies of the GNU General Public License and the
+GNU Lesser General Public License along with the PaGMO library.  If not,
+see https://www.gnu.org/licenses/. */
+
+#ifndef PAGMO_ALGORITHMS_GAC0_HPP
```

Comment:
Typo? PAGMO_ALGORITHMS_GAC0_HPP with an O not a zero.

Reply:
Yes, typo

Reply:
Do not resolve conversation before commiting the solution.

Reply:
I just saw this comment: Get rid of the folder tuDATbundle, move your files into the pagmo structure

In which folder do you want me to move them?

Reply:
@Sceki 

1) Move the hpp file in root_folder//include/pagmo/algorithms (where all algorithms are)
2) Move the cpp test file in root_folder/tests (where all tests are)
3) Add your test to the CMakeLists.txt
4) Build the test.
5) Commit your code
6) Wait for us to format and check.

Reply:
I just committed the gaco_rosenbrock.cpp, GACO.hpp and CMakeLists.txt files moving them into the (hopefully) right folders. I hope I executed the procedures correctly

Reply:
I will send a PR to your repo with fixes. Please study them accurately as to avoid too many iterations.